From b6df1d6f014462663204f5feee4304953024e8a0 Mon Sep 17 00:00:00 2001
From: Jenkins <dev-kitchen@cloudera.com>
Date: Tue, 28 Nov 2017 06:06:57 -0800
Subject: [PATCH 1322/1363] Branching for 5.14.1-SNAPSHOT on Tue Nov 28
 06:05:46 PST 2017

JOB_NAME : 'Cut-Release-Branches'
BUILD_NUMBER : '514'
CODE_BRANCH : ''
OLD_CDH_BRANCH : 'cdh5'

Pushed to remote apache	git@github.sf.cloudera.com:Mirrors/apache-hive.git (push)
origin	git@github.sf.cloudera.com:CDH/hive.git (push)
---
 accumulo-handler/pom.xml                        |    2 +-
 ant/pom.xml                                     |    2 +-
 beeline/pom.xml                                 |    2 +-
 classification/pom.xml                          |    2 +-
 cli/pom.xml                                     |    2 +-
 common/pom.xml                                  |    2 +-
 contrib/pom.xml                                 |    2 +-
 hbase-handler/pom.xml                           |    2 +-
 hcatalog/core/pom.xml                           |    2 +-
 hcatalog/hcatalog-pig-adapter/pom.xml           |    2 +-
 hcatalog/pom.xml                                |    2 +-
 hcatalog/server-extensions/pom.xml              |    2 +-
 hcatalog/streaming/pom.xml                      |    2 +-
 hcatalog/webhcat/java-client/pom.xml            |    2 +-
 hcatalog/webhcat/svr/pom.xml                    |    2 +-
 hwi/pom.xml                                     |    2 +-
 itests/custom-serde/pom.xml                     |    2 +-
 itests/custom-udfs/pom.xml                      |    2 +-
 itests/custom-udfs/udf-classloader-udf1/pom.xml |    2 +-
 itests/custom-udfs/udf-classloader-udf2/pom.xml |    2 +-
 itests/custom-udfs/udf-classloader-util/pom.xml |    2 +-
 itests/hcatalog-unit/pom.xml                    |    2 +-
 itests/hive-blobstore/pom.xml                   |    2 +-
 itests/hive-jmh/pom.xml                         |    2 +-
 itests/hive-minikdc/pom.xml                     |    2 +-
 itests/hive-unit-hadoop2/pom.xml                |    2 +-
 itests/hive-unit/pom.xml                        |    2 +-
 itests/pom.xml                                  |    2 +-
 itests/qtest-spark/pom.xml                      |    2 +-
 itests/qtest/pom.xml                            |    2 +-
 itests/test-serde/pom.xml                       |    2 +-
 itests/util/pom.xml                             |    2 +-
 jdbc/pom.xml                                    |    2 +-
 metastore/pom.xml                               |    2 +-
 odbc/pom.xml                                    |    2 +-
 packaging/pom.xml                               |    2 +-
 pom.xml                                         |    6 +++---
 ql/pom.xml                                      |    2 +-
 serde/pom.xml                                   |    2 +-
 service/pom.xml                                 |    2 +-
 shims/0.20S/pom.xml                             |    2 +-
 shims/0.23/pom.xml                              |    2 +-
 shims/aggregator/pom.xml                        |    2 +-
 shims/common/pom.xml                            |    2 +-
 shims/pom.xml                                   |    2 +-
 shims/scheduler/pom.xml                         |    2 +-
 spark-client/pom.xml                            |    2 +-
 testutils/pom.xml                               |    2 +-
 48 files changed, 50 insertions(+), 50 deletions(-)

diff --git a/accumulo-handler/pom.xml b/accumulo-handler/pom.xml
index b9af689..c0674cc 100644
--- a/accumulo-handler/pom.xml
+++ b/accumulo-handler/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/ant/pom.xml b/ant/pom.xml
index 8d20107..e043802 100644
--- a/ant/pom.xml
+++ b/ant/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/beeline/pom.xml b/beeline/pom.xml
index cf79190..bea587e 100644
--- a/beeline/pom.xml
+++ b/beeline/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/classification/pom.xml b/classification/pom.xml
index 7e866ae..f52813d 100644
--- a/classification/pom.xml
+++ b/classification/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/cli/pom.xml b/cli/pom.xml
index d2dcc6b..7a1b4a2 100644
--- a/cli/pom.xml
+++ b/cli/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/common/pom.xml b/common/pom.xml
index f99b3d2..2c739eb 100644
--- a/common/pom.xml
+++ b/common/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/contrib/pom.xml b/contrib/pom.xml
index d0b7ee9..8b56cbd 100644
--- a/contrib/pom.xml
+++ b/contrib/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hbase-handler/pom.xml b/hbase-handler/pom.xml
index cb7a2f2..d9513e7 100644
--- a/hbase-handler/pom.xml
+++ b/hbase-handler/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/core/pom.xml b/hcatalog/core/pom.xml
index 7e93df8..f9dd8c2 100644
--- a/hcatalog/core/pom.xml
+++ b/hcatalog/core/pom.xml
@@ -26,7 +26,7 @@
   <parent>
     <groupId>org.apache.hive.hcatalog</groupId>
     <artifactId>hive-hcatalog</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/hcatalog-pig-adapter/pom.xml b/hcatalog/hcatalog-pig-adapter/pom.xml
index 703ed14..908408f 100644
--- a/hcatalog/hcatalog-pig-adapter/pom.xml
+++ b/hcatalog/hcatalog-pig-adapter/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <groupId>org.apache.hive.hcatalog</groupId>
     <artifactId>hive-hcatalog</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/pom.xml b/hcatalog/pom.xml
index 8aa86c6..52ac501 100644
--- a/hcatalog/pom.xml
+++ b/hcatalog/pom.xml
@@ -24,7 +24,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/server-extensions/pom.xml b/hcatalog/server-extensions/pom.xml
index 83a3a5b..cef4579 100644
--- a/hcatalog/server-extensions/pom.xml
+++ b/hcatalog/server-extensions/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <groupId>org.apache.hive.hcatalog</groupId>
     <artifactId>hive-hcatalog</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/streaming/pom.xml b/hcatalog/streaming/pom.xml
index d6064a7..25cce39 100644
--- a/hcatalog/streaming/pom.xml
+++ b/hcatalog/streaming/pom.xml
@@ -20,7 +20,7 @@
   <parent>
     <groupId>org.apache.hive.hcatalog</groupId>
     <artifactId>hive-hcatalog</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/webhcat/java-client/pom.xml b/hcatalog/webhcat/java-client/pom.xml
index b0e3ce6..b86dc06 100644
--- a/hcatalog/webhcat/java-client/pom.xml
+++ b/hcatalog/webhcat/java-client/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <groupId>org.apache.hive.hcatalog</groupId>
     <artifactId>hive-hcatalog</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/hcatalog/webhcat/svr/pom.xml b/hcatalog/webhcat/svr/pom.xml
index ae65a58..fedb291 100644
--- a/hcatalog/webhcat/svr/pom.xml
+++ b/hcatalog/webhcat/svr/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <groupId>org.apache.hive.hcatalog</groupId>
     <artifactId>hive-hcatalog</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/hwi/pom.xml b/hwi/pom.xml
index fd7d792..00854b1 100644
--- a/hwi/pom.xml
+++ b/hwi/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/custom-serde/pom.xml b/itests/custom-serde/pom.xml
index dedfc65..e42e1cf 100644
--- a/itests/custom-serde/pom.xml
+++ b/itests/custom-serde/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/custom-udfs/pom.xml b/itests/custom-udfs/pom.xml
index befbc7a..cbb22bc 100644
--- a/itests/custom-udfs/pom.xml
+++ b/itests/custom-udfs/pom.xml
@@ -19,7 +19,7 @@ limitations under the License.
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/custom-udfs/udf-classloader-udf1/pom.xml b/itests/custom-udfs/udf-classloader-udf1/pom.xml
index 1150e7d..9a175d5 100644
--- a/itests/custom-udfs/udf-classloader-udf1/pom.xml
+++ b/itests/custom-udfs/udf-classloader-udf1/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it-custom-udfs</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/custom-udfs/udf-classloader-udf2/pom.xml b/itests/custom-udfs/udf-classloader-udf2/pom.xml
index fdfca94..8129c4d 100644
--- a/itests/custom-udfs/udf-classloader-udf2/pom.xml
+++ b/itests/custom-udfs/udf-classloader-udf2/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it-custom-udfs</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/custom-udfs/udf-classloader-util/pom.xml b/itests/custom-udfs/udf-classloader-util/pom.xml
index cf2648e..f3f5a8b 100644
--- a/itests/custom-udfs/udf-classloader-util/pom.xml
+++ b/itests/custom-udfs/udf-classloader-util/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it-custom-udfs</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/hcatalog-unit/pom.xml b/itests/hcatalog-unit/pom.xml
index 606150f..7db85b0 100644
--- a/itests/hcatalog-unit/pom.xml
+++ b/itests/hcatalog-unit/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/hive-blobstore/pom.xml b/itests/hive-blobstore/pom.xml
index 1d70bd2..6624abe 100644
--- a/itests/hive-blobstore/pom.xml
+++ b/itests/hive-blobstore/pom.xml
@@ -20,7 +20,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/hive-jmh/pom.xml b/itests/hive-jmh/pom.xml
index 32a4326..db5da2f 100644
--- a/itests/hive-jmh/pom.xml
+++ b/itests/hive-jmh/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/hive-minikdc/pom.xml b/itests/hive-minikdc/pom.xml
index be620b0a0..df1e101 100644
--- a/itests/hive-minikdc/pom.xml
+++ b/itests/hive-minikdc/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/hive-unit-hadoop2/pom.xml b/itests/hive-unit-hadoop2/pom.xml
index eef94da..3572c82 100644
--- a/itests/hive-unit-hadoop2/pom.xml
+++ b/itests/hive-unit-hadoop2/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/hive-unit/pom.xml b/itests/hive-unit/pom.xml
index c1e0f52..6fb01dc 100644
--- a/itests/hive-unit/pom.xml
+++ b/itests/hive-unit/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/pom.xml b/itests/pom.xml
index e2eb74e..9ab690b 100644
--- a/itests/pom.xml
+++ b/itests/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/qtest-spark/pom.xml b/itests/qtest-spark/pom.xml
index 6ec28f8..6b9611c 100644
--- a/itests/qtest-spark/pom.xml
+++ b/itests/qtest-spark/pom.xml
@@ -20,7 +20,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/qtest/pom.xml b/itests/qtest/pom.xml
index 4916693..4805dd3 100644
--- a/itests/qtest/pom.xml
+++ b/itests/qtest/pom.xml
@@ -20,7 +20,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/test-serde/pom.xml b/itests/test-serde/pom.xml
index b5fb383..db1ae44 100644
--- a/itests/test-serde/pom.xml
+++ b/itests/test-serde/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/itests/util/pom.xml b/itests/util/pom.xml
index 2fb1a15..1e43146 100644
--- a/itests/util/pom.xml
+++ b/itests/util/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive-it</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/jdbc/pom.xml b/jdbc/pom.xml
index b0982e2..72fe36a 100644
--- a/jdbc/pom.xml
+++ b/jdbc/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/metastore/pom.xml b/metastore/pom.xml
index 15fbfb7..032d03f 100644
--- a/metastore/pom.xml
+++ b/metastore/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/odbc/pom.xml b/odbc/pom.xml
index 4c96474..a6fef5fd 100644
--- a/odbc/pom.xml
+++ b/odbc/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/packaging/pom.xml b/packaging/pom.xml
index 23e2e26..08b2738 100644
--- a/packaging/pom.xml
+++ b/packaging/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/pom.xml b/pom.xml
index 287194f..1f91071 100644
--- a/pom.xml
+++ b/pom.xml
@@ -17,12 +17,12 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>cdh-root</artifactId>
-    <version>5.14.0-SNAPSHOT</version>
+    <version>5.14.1-SNAPSHOT</version>
   </parent>
 
   <groupId>org.apache.hive</groupId>
   <artifactId>hive</artifactId>
-  <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+  <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
   <packaging>pom</packaging>
 
   <name>Hive</name>
@@ -55,7 +55,7 @@
   </modules>
 
   <properties>
-    <cdh.hive.version>1.1.0-cdh5.14.0-SNAPSHOT</cdh.hive.version>
+    <cdh.hive.version>1.1.0-cdh5.14.1-SNAPSHOT</cdh.hive.version>
     <hive.version.shortname>1.1.0</hive.version.shortname>
 
     <!-- Build Properties -->
diff --git a/ql/pom.xml b/ql/pom.xml
index e77f5e2..465a43a 100644
--- a/ql/pom.xml
+++ b/ql/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/serde/pom.xml b/serde/pom.xml
index 897462d..8a81e39 100644
--- a/serde/pom.xml
+++ b/serde/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/service/pom.xml b/service/pom.xml
index 4690936..e2b27df 100644
--- a/service/pom.xml
+++ b/service/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/shims/0.20S/pom.xml b/shims/0.20S/pom.xml
index 0b12bb3..fa996c7 100644
--- a/shims/0.20S/pom.xml
+++ b/shims/0.20S/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/shims/0.23/pom.xml b/shims/0.23/pom.xml
index 49196f6..f5f8b05 100644
--- a/shims/0.23/pom.xml
+++ b/shims/0.23/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/shims/aggregator/pom.xml b/shims/aggregator/pom.xml
index e2cbe74..9dcffca 100644
--- a/shims/aggregator/pom.xml
+++ b/shims/aggregator/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/shims/common/pom.xml b/shims/common/pom.xml
index 73463c9..6f787eb 100644
--- a/shims/common/pom.xml
+++ b/shims/common/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/shims/pom.xml b/shims/pom.xml
index 65b8218..db60251 100644
--- a/shims/pom.xml
+++ b/shims/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
diff --git a/shims/scheduler/pom.xml b/shims/scheduler/pom.xml
index 595e697..68c6cd6 100644
--- a/shims/scheduler/pom.xml
+++ b/shims/scheduler/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
diff --git a/spark-client/pom.xml b/spark-client/pom.xml
index 4a3808a..7b20a1d 100644
--- a/spark-client/pom.xml
+++ b/spark-client/pom.xml
@@ -22,7 +22,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
   </parent>
 
   <groupId>org.apache.hive</groupId>
diff --git a/testutils/pom.xml b/testutils/pom.xml
index f6b1a5e..ec6baea 100644
--- a/testutils/pom.xml
+++ b/testutils/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hive</groupId>
     <artifactId>hive</artifactId>
-    <version>1.1.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.1.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
-- 
1.7.9.5

